### ISSUE TITLE
Mac M1/M2/M3 Processor

### ISSUE THREAD

**Issue by ilsaul** [optyfr/JRomManager#282]
Hi,
Mac before use Intel processor and now use arc processor.
On my mac the version 3.0.4 don't work but the old 3.0.0 work.

3.0.4 has the error below:
`dlopen(/Users/XXXX/Applications/JRomManager.app/Contents/runtime/Contents/Home/lib/libjli.dylib) failed. Error: dlopen(/Users/XXXX/Applications/JRomManager.app/Contents/runtime/Contents/Home/lib/libjli.dylib, 0x0001): tried: '/Users/XXXX/Applications/JRomManager.app/Contents/runtime/Contents/Home/lib/libjli.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/XXXX/Applications/JRomManager.app/Contents/runtime/Contents/Home/lib/libjli.dylib' (no such file), '/Users/XXXX/Applications/JRomManager.app/Contents/runtime/Contents/Home/lib/libjli.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e' or 'arm64'))`

I tried to understand the problem and the first thing I found is the jdk in build.gradle which is missing the arc architecture:
```
targetPlatform("macos-x64") {
		jdkHome = jdkDownload("https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.3%2B9/OpenJDK21U-jdk_x64_mac_hotspot_21.0.3_9.tar.gz")
}
```
but x86 is intel.
There is the version for arc:
OpenJDK21U-jdk_aarch64_mac_hotspot_21.0.3_9.tar.gz

**Comment by optyfr** [comment]
I think I have found a problem due to a github action images name stupid move : "macos-latest" was about macos-12 for intel before, and now it is about macos-14 for M1 !
A a result this will build an image for M1 but with intel jdk :-(

**Comment by optyfr** [comment]
should solved with commit f50fdd5
